pkg/cmd/server/server.go

```diff
@@ -217,6 +224,12 @@ func (c *Config) Complete() (RunnableServer, error) {
 		v1SchemaServiceOption = services.V1SchemaServiceDisabled
 	}
 
+	watchServiceOption := services.WatchServiceEnabled
+	if !datastoreFeatures.CanWatch {
+		log.Warn().Msg("watch api disabled; underlying datastore does not support it")
```

Comment:
I wonder if we can at least print out a link to how to enable it? Ideally, we'd provide the command to run to enable it

Reply:
I debated adding a `CanWatchMessage string` to the `Features` object so that the datastore could return more info about what to do, but it seemed like overkill since this is something we can easily fill with docs

Reply:
Maybe have the Features object contain a helper text field, and we put a link to the docs in there?

Reply:
Added a per-feature message field that we can add help text to